.github/workflows/confidence.yml

```diff
@@ -10,10 +10,10 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.6', '3.7', '3.8', '3.9']
+        python-version: ['3.8', '3.9', '3.10']
```

Comment:
What's the reason for dropping 3.7 here?

Reply:
Python 3.7 is reaching end of life in June, so I just thought we can stopp running tests on it already.